Revit_Core_Engine/Query/PipeSectionProperty.cs

```diff
@@ -38,23 +38,23 @@ public static partial class Query
         /****               Public Methods              ****/
         /***************************************************/
 
-        [Description("Pipe section property converted fom Revit.")]
-        [Input("pipe", "Revit pipe type to be converted to a section property.")]
+        [Description("Convert a Revit pipe into a BHoM pipe section property.")]
+        [Input("pipe", "Revit pipe to be converted into a BHoM section property.")]
         [Input("settings", "Revit settings.")]
         [Input("refObjects", "Referenced objects.")]
-        [Output("property", "BHoM duct section property.")]
+        [Output("sectionProperty", "BHoM pipe section property converted from a Revit pipe.")]
```

Comment:
It is not conversion, rather query/extraction.

Reply:
Revised accordingly